pymatgen/io/lobster/outputs.py

```diff
@@ -499,8 +499,8 @@ def __init__(
                 are_coops=are_coops,
                 are_cobis=are_cobis,
                 list_labels=labels,
-                list_atom1=atoms1,
-                list_atom2=atoms2,
+                list_atom1=atom1s,
```

Comment:
Why `atom1s`? I am a bit confused by this nomenclature

Reply:
Thanks for asking. `atom1s` is the collection of `atom1`, but indeed... It feels a bit weird 🙈

Reply:
Yeah. It reminds me of 1s orbitals which might be very confusing here.

Reply:
Sorry for the confusion, but it should be fine under the context:
```python
labels: list[str] = []
atom1s: list[str] = []
atom2s: list[str] = []
lens: list[float] = []
translations: list[Tuple3Ints] = []
nums: list[int] = []
icohps: list[dict[Spin, float]] = []
```

`atom1s` looks more consistent with its sibling collections in this context :)